helm/conjur-config-cluster-prep/generated/conjur-config-cluster-prep.yaml

```diff
@@ -0,0 +1,258 @@
+---
+# Source: conjur-config-cluster-prep/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: conjur-serviceaccount
+  labels:
+    release: cluster-prep
+    heritage: Helm
+    app.kubernetes.io/name: "conjur-serviceaccount"
+    app.kubernetes.io/component: "conjur-kubernetes-identity"
+    app.kubernetes.io/instance: "conjur-serviceaccount"
+    app.kubernetes.io/part-of: "conjur-config"
+    conjur.org/name: "conjur-serviceaccount"
+    helm.sh/chart: conjur-config-cluster-prep-0.1.0
+---
+# Source: conjur-config-cluster-prep/templates/golden_configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: conjur-configmap
+    labels:
+      release: cluster-prep
+      heritage: Helm
+      app.kubernetes.io/name: "conjur-golden-configmap"
+      app.kubernetes.io/component: "conjur-reference-config"
+      app.kubernetes.io/instance: "conjur-golden-configmap"
+      app.kubernetes.io/part-of: "conjur-config"
+      conjur.org/name: "conjur-golden-configmap"
+      helm.sh/chart: conjur-config-cluster-prep-0.1.0
+data:
+    # authn-k8s Configuration 
+    authnK8sAuthenticatorID: myAuthenticatorId
+    authnK8sClusterRole: conjur-clusterrole
+    authnK8sNamespace: default
+    authnK8sServiceAccount: conjur-serviceaccount
+
+    # Conjur Configuration 
+    conjurAccount: myConjurAccount
+    conjurApplianceUrl: https://conjur-oss.conjur-oss.svc.cluster.local
+    conjurSslCertificate: ""
+    conjurSslCertificateBase64: ""
+---
+# Source: conjur-config-cluster-prep/templates/tests/test-cluster-prep-configmap.yaml
```

Comment:
I believe this can be addressed by adding the test hook annotations to the test ConfigMap and RoleBinding, similar to what we have for the test Pod:

```
  annotations:
    "helm.sh/hook": test-success
```

Reply:
Done - used `success` instead of `test-success` based on latest Helm v3 guidance